src/node/history.h

```diff
@@ -749,27 +743,26 @@ namespace ccf
           fmt::format("No endorsed certificate set to emit signature"));
       }
 
-      // NB: This call will also reset consensus' should_sign, so we should
-      // always proceed to producing a signature from here.
-      auto commit_txid = consensus->get_signable_txid();
+      auto previous_signature_seqno =
+        consensus->get_previous_committable_seqno();
       auto txid = store.next_txid();
 
-      last_signed_tx = commit_txid.version;
-      time_of_last_signature =
-        threading::ThreadMessaging::thread_messaging.get_current_time_offset();
-
       LOG_DEBUG_FMT(
-        "Signed at {} in view: {} commit was: {}.{} (previous .{})",
+        "Signed at {} in view: {}, previous signature was at {}",
         txid.version,
         txid.term,
-        commit_txid.term,
-        commit_txid.version,
-        commit_txid.previous_version);
+        previous_signature_seqno);
 
       store.commit(
         txid,
         std::make_unique<MerkleTreeHistoryPendingTx<T>>(
-          txid, commit_txid, store, *this, id, kp, endorsed_cert.value()),
+          txid,
+          previous_signature_seqno,
```

Comment:
Why are we passing the previous signature seqno? I thought we'd agreed to zero this out?